dds_web/api/user.py

```diff
@@ -1334,6 +1334,16 @@ def row_to_dict(entry) -> dict:
                 hit["Unit"] = hit["Unit"].name
             return hit
 
+        def mark_if_owner(entry, invite_id):
+            """Given a list of users to print, If the researcher is Project Owner, list the role as Owner."""
```

Comment:
```suggestion
            """Given an invite for printing, If the researcher is Project Owner, list the role as Owner."""
```
since `entry` is a dict for a single invite